Sources/BrowserServicesKit/ContentBlocking/AdClickAttribution/AdClickAttributionLogic.swift

```diff
@@ -216,8 +224,9 @@ public class AdClickAttributionLogic {
         eventReporting?.fire(.adAttributionActive)
         registerFirstActivity = false
     }
-    
+
     private func disableAttribution() {
+        cancelAttributedRulesCompilationIfNeeded()
```

Comment:
This is called only if state is active, so I don't think it does anything either.